ui/admin/app/components/form/role/grants/index.hbs

```diff
@@ -13,20 +13,18 @@
     <Rose::List::KeyValue as |list|>
       {{! Add grant field (only one per form) }}
       <list.item as |item|>
-        <form.input
-          @contextual={{true}}
-          @name='grant'
-          @type='text'
-          @value={{this.newGrantString}}
-          as |input|
-        >
+        <form.input @contextual={{true}} as |input|>
           <item.key>
             <Hds::Form::Label>
               {{t 'resources.role.grant.actions.create'}}
             </Hds::Form::Label>
           </item.key>
           <item.cell>
-            <input.field title={{t 'form.grant.help'}} />
+            <input.field
+              type='text'
```

Comment:
Note: `<item.cell>` will not accept an Hds form field component. It either breaks the form, or breaks the `Add` button. I wonder if we should refactor this file and remove `Rose::List::KeyValue` because it does not seem to respond well to mixing Hds components within. Open to suggestions. Still WIP.